### PR TITLE
[SOC-5485] Refused Spaces suggestions should not be displayed again

### PR DESCRIPTION
**Pull request by AtiaZ**
fix description : 

1- add isIgnored() to verify if the space is ignored by user or not.
2- fix the suggest.js file when ignoring spaces.
3- fix unit test.

### CODE REVIEW

File: component/common/src/main/java/org/exoplatform/platform/common/rest/services/SuggestSpacesPortlet/SpaceRestServices.java [hunk 1]
```diff
@@ -82,16 +83,18 @@ public Response getSuggestions(@Context SecurityContext sc, @Context UriInfo uri
...
             IdentityManager identityManager = (IdentityManager) container.getComponentInstanceOfType(IdentityManager.class);
+            RelationshipManager relationshipManager = (RelationshipManager) container.getComponentInstanceOfType(RelationshipManager.class);
             Identity identity = identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, userId, false);
-            ListAccess<Identity> connectionsLA = identityManager.getConnectionsWithListAccess(identity);
+            ListAccess<Identity> connectionsLA = relationshipManager.getConnections(identity);
             
             final Map<Space, Integer> spacesWithMemberNum = new HashMap<Space, Integer>();
             int maxConnectionsToLoad = 100;
-            int maxSpacesToLoad = 50;
+            int maxSpacesToLoad = 20;
```
Comment (thomasdelhomenie) [inline component/common/src/main/java/org/exoplatform/platform/common/rest/services/SuggestSpacesPortlet/SpaceRestServices.java]: Why did you change this value ?

Comment (AtiaZ) [inline component/common/src/main/java/org/exoplatform/platform/common/rest/services/SuggestSpacesPortlet/SpaceRestServices.java]: we don't need to load 50 spaces since we will display only 10. Also, we have 2 spaces only displayed in the gadget.

Comment (thomasdelhomenie) [inline component/common/src/main/java/org/exoplatform/platform/common/rest/services/SuggestSpacesPortlet/SpaceRestServices.java]: I guess it was a workaround to be sure to return at least 10 spaces. The list of retrieved is filtered (hidden, closed,  already member, ...), so even if we load 50 spaces, we could end up with 0 suggested spaces if the 50 spaces are hidden for example.

Comment (AtiaZ) [inline component/common/src/main/java/org/exoplatform/platform/common/rest/services/SuggestSpacesPortlet/SpaceRestServices.java]: change reverted.

**Comment by AtiaZ** [comment]
Merged in develop.
